pom.xml

```diff
@@ -58,11 +58,11 @@
     <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
     <!-- version properties -->
     <avro.version>1.7.7</avro.version>
-    <cdap.version>6.8.0-SNAPSHOT</cdap.version>
+    <cdap.version>6.8.0</cdap.version>
     <junit.version>4.12</junit.version>
     <commons.io.version>2.6</commons.io.version>
     <commons.lang.version>2.6</commons.lang.version>
-    <hadoop.version>2.3.0</hadoop.version>
+    <hadoop.version>2.10.2</hadoop.version>
     <spark2.version>2.1.3</spark2.version>
     <hydrator.version>2.10.0-SNAPSHOT</hydrator.version>
```

Comment:
Should this be `2.10.0` instead of `2.10.0-SNAPSHOT`?

Reply:
Fixed, thanks!